### DISCUSSION TITLE
RSSI calibration

### DISCUSSION THREAD

**Discussion by hendpraz88** [egzumer/uv-k5-firmware-custom#72] [General]
I always follow your firmware update. your firmware is very great. but, It seems like the wrong calibration from the RSSI calibration from the firmware v0.14.1 to the newest version. I read in the https://hamwaves.com/decibel/en/ There is an explanation of the S-meter for HF and VHF/UHF. so, I think the RSSI  calibration currently used in the newest firmware is calibration for the HF (high frequency) band. for example, I have noticed in my UVK5 that the S9 signal bar value is -73 dBm.  but in the S-meter VHF/UHF the value of the S9 signal is -93 dBm. So, maybe you can change and update the RSSI calibration for S-meter VHF/UHF. Thank you.

**Comment by egzumer** [comment]
Thanks for reporting, there is a bug indeed, although it is only for values greater than S9. S1-S9 is calculated from S1=-141 dBm in 6dB steps.
